### PR TITLE
docker-compose: Env vars for custom storage paths

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: images/compose/README.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ mkdir ../../pgsql ../../pulp_storage
 podman unshare chown 700:700 ../../pulp_storage
 podman-compose -f docker-compose.folders.yml up
 ```
+You can specify your own paths with:
+
+```
+PULP_STORAGE_PATH="./storage" PULP_DATABASE_PATH="./database" podman-compose -f docker-compose.folders.yml up
```
Comment (mikedep333) [inline images/compose/README.md]: Document PULP_POSTGRES_PATH too?

**Review by mikedep333 — CHANGES_REQUESTED** [review]

File: images/compose/README.md (hunk 1 above)
Comment (jlsherrill) [reply images/compose/README.md]: documented my own option incorrectly :)

Comment (jlsherrill) [reply images/compose/README.md]: fixed

**Comment by jlsherrill** [comment]
as part of our integration, we don't actually need this anymore.  Feel free to close this or merge it :)

**Comment by mikedep333** [comment]
Thank you, but I honestly don't think this is needed. I doubt fixed folder names will be a problem for users.

**Pull request closed without merging**